plugins/hypervisors/vmware/src/com/cloud/storage/resource/VmwareStorageSubsystemCommandHandler.java

```diff
@@ -61,11 +61,28 @@ public void setStorageManager(VmwareStorageManager storageManager) {
         this.storageManager = storageManager;
     }
 
-    public VmwareStorageSubsystemCommandHandler(StorageProcessor processor, String nfsVersion) {
+    public VmwareStorageSubsystemCommandHandler(StorageProcessor processor, Integer nfsVersion) {
         super(processor);
         this._nfsVersion = nfsVersion;
     }
 
+    /**
+     * Reconfigure NFS version for storage operations
+     * @param nfsVersion NFS version to set
+     * @return true if NFS version coulb de configured, false in other case
```

Comment:
Should it be "_true if NFS version coul**d b**e configured, false in other case_"? Thanks for the javadoc, good work.